.github/workflows/ci.yml

```diff
@@ -282,11 +282,12 @@ jobs:
     strategy:
```

Comment:
Can we add a timeout on this job, and name the job using the check name?
(Maybe this is already done in other PRs?)